snake_python/libs/snake.py

```diff
@@ -71,7 +72,7 @@ def update(self) -> None:
             for segment in self.segments:
                 segment.update()
 
-            if int(self.map[self.x][self.y]) > int(time()) and not self.isGuest(): # this is not working, please fix
+            if self.getHead().x == 0 or self.getHead() == self.screen.get_width() or self.getHead().y == 0 or self.getHead().y == self.screen.get_height() and not self.isGuest():
```

Comment:
We could implement a flag to enable wrapping around the screen. I will open a issue / feature request for that

Reply:
sounds good